# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/interfaces/base/nsIBrowserChild.idl
# Commit: 405100db6c45
# Full Hash: 405100db6c45f0463286067df5b491f74790d9aa
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2019-07-11 09:51:12
# Regressor Bug: 1543251
# File Overlap Count: 1
# Description:
#   Bug 1543251 - Move hasBeforeUnload from PBrowser to PWindowGlobal, r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D37003
# ==============================================================================

diff -r 0c408343080d -r 405100db6c45 dom/interfaces/base/nsIBrowserChild.idl
--- a/dom/interfaces/base/nsIBrowserChild.idl	Thu Jul 04 07:41:57 2019 +0000
+++ b/dom/interfaces/base/nsIBrowserChild.idl	Wed Jul 10 21:13:44 2019 +0000
@@ -38,9 +38,6 @@
    */
   attribute boolean hasSiblings;
 
-  [noscript, notxpcom] void beforeUnloadAdded();
-  [noscript, notxpcom] void beforeUnloadRemoved();
-
   /**
    * Tell the nsIBrowserChild that it should begin sending its nsIWebProgress
    * events to its nsIBrowserParent.